install.rst

```diff
@@ -316,6 +317,11 @@ build time:
     * krb5 libraries and headers
     * ipsumdump (for trace-summary; https://github.com/kohler/ipsumdump)
 
+ZeroMQ (e.g., libzmq3-dev on Debian/Ubuntu or cppzmq-devel on Fedora) is a
+requirement for developers working on core Zeek as some of Zeek's central
+tests require it to be available. Otherwise, for operating Zeek, ZeroMQ
+is optional unless you want to test with the :ref:`cluster_backend_zeromq`.
```

Comment:
```suggestion
is optional unless you want to test with :ref:`cluster_backend_zeromq`.
```

Reply:
I think the "the" needs to be there given the title of that page :-)

![image](https://github.com/user-attachments/assets/f32e7804-4e66-48e2-90eb-7edfeba7833b)


Feel free to fix it up if you still disagree.